app-emulation/docker-runc/docker-runc-1.0.1.ebuild

```diff
@@ -11,8 +11,8 @@ COREOS_GO_VERSION="go1.13"
 # Update the patch number when this commit is changed (i.e. the _p in the ebuild).
 # The patch version is arbitrarily the number of commits since the tag version
 # specified in the ebuild name. For example:
-# $ git log --oneline v1.0.0..${COMMIT_ID} | wc -l
-COMMIT_ID="60cd6da6ef9fbf689682922617efff01c5f221c0"
+# $ git log --oneline v1.0.1..${COMMIT_ID} | wc -l
```

Comment:
The comment about updating the patch number seems to be outdated? Not sure how relevant it is now.

Reply:
Right, I removed the obsolete comments.